confgenerator/fluentbit/modify.go

```diff
@@ -0,0 +1,96 @@
+package fluentbit
+
+import "fmt"
+
+// ModifyRule is a string corresponding to one of the Configuration Rules of fluentbit
+// for a full list please refer to https://docs.fluentbit.io/manual/pipeline/filters/modify#configuration-parameters
+type ModifyRule = string
```

Comment:
`= string` causes these to be type aliases instead of new types; you probably meant to say `type ModifyRule string`.